client/.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
```

Comment:
I want to clean this up myself in a new PR, which I will merge into @samuko-things.
So I'll approve this PR so it'll let me merge into that branch.